server/server.js

```diff
@@ -220,11 +234,22 @@ Server.prototype._handleResourceMessage = function(socket, message, messageType)
     this._storeResource(resource);
     this._persistenceSubscribe(resource.name, socket.id);
     this._updateLimits(socket, message, resource.options);
+    this._stampMessage(socket, message);
```

Comment:
It has nothing to do with Sentry refactor. Overall, this PR is too big has some irrelevant changes which make it difficult to review already.